### PR TITLE
ENH: Add SSP joint plots to reports

### PR DESCRIPTION
**Pull request by larsoner**
### Before merging …

- [x] Changelog has been updated (`docs/source/changes.md`)

Locally at least it produces something reasonable:

Screenshot from 2022-09-28 15-59-55

### CODE REVIEW

**Comment by hoechenberger** [comment]
Nice! Can we change the order such that the topoplot comes first?

**Comment by larsoner** [comment]
You mean horizontally? I don't think we should here. The order comes from the upstream joint plot function where we chose the order presumably with consensus and review already

**Comment by larsoner** [comment]
@hoechenberger how do we resolve:

    RuntimeError: Error 403 when trying to download
    /home/circleci/mne_data/ds004229/sub-102/meg/sub-102_task-amnoise_meg.fif from
    https://s3.amazonaws.com/openneuro.org/ds004229/sub-102/meg/sub-102_task-amnoise_meg.fif?versionId=Mfnz6drCpmRBzW1NlZA7C4o5.ZCQjWae

File: .circleci/config.yml [hunk 1]
```diff
@@ -942,47 +949,6 @@ jobs:
...
-            - data-cache-ds003392-{{ .Revision }}
-      - restore_cache:
-          keys:
-            - data-cache-ds004229-{{ .Revision }}
-      - restore_cache:
-          keys:
-            - data-cache-eeg_matchingpennies-{{ .Revision }}
-      - restore_cache:
-          keys:
-            - data-cache-ERP_CORE-{{ .Revision }}
```
Comment (larsoner) [inline .circleci/config.yml:985]: I expect these to break things -- I plan to fix them here

**Comment by hoechenberger** [comment]
> You mean horizontally? I don't think we should here. The order comes from the upstream joint plot function where we chose the order presumably with consensus and review already

Oh i didn't realize this came from upstream, I actually didn't look at your code before writing that comment! Let's keep it the way it is then.

**Comment by hoechenberger** [comment]
> @hoechenberger how do we resolve:
> 
> 
> 
>     RuntimeError: Error 403 when trying to download
> 
>     /home/circleci/mne_data/ds004229/sub-102/meg/sub-102_task-amnoise_meg.fif from
> 
>     https://s3.amazonaws.com/openneuro.org/ds004229/sub-102/meg/sub-102_task-amnoise_meg.fif?versionId=Mfnz6drCpmRBzW1NlZA7C4o5.ZCQjWae
> 
> 
> 
> 


If it persists we need to open an issue with OpenNeuro. Can you ping me if the problem doesn't go away in its own? I can then take over.

**Comment by larsoner** [comment]
> Can you ping me if the problem doesn't go away in its own? I can then take over.

It has been like this through 4 or 5 tries over about 24 hours. It's that long enough?

**Comment by hoechenberger** [comment]
> > Can you ping me if the problem doesn't go away in its own? I can then take over.
> 
> 
> 
> It has been like this through 4 or 5 tries over about 24 hours. It's that long enough?

Ouch. Yes. I will try to take a look.

**Comment by hoechenberger** [comment]
@larsoner I've opened a bug report: https://github.com/OpenNeuroOrg/openneuro/issues/2677

File: .circleci/config.yml (hunk 1 above)
Comment (agramfort) [reply .circleci/config.yml:985]: Did you actually break something?

**Review by agramfort — APPROVED** [review]
If nothing breaks. Lets merge ! 🙏 @larsoner

**Comment by hoechenberger** [comment]
I'd also say it's okay to move ahead here and merge if @larsoner is okay with this. It might take another day or two for the OpenNeuro issue to get resolved…

**Comment by larsoner** [comment]
We can wait. I want to fix the thing I likely broke on CircleCI here, too

**Comment by larsoner** [comment]
For now I'm just going to disable ds004229. That way we can get green CIs on these PRs and merge.

**Pull request merged**